src/components/network/PageNetworks.vue

```diff
@@ -57,14 +57,14 @@ export default {
     ...mapState(["session"]),
     ...mapGetters([`networks`]),
     mainNetworks() {
-      return this.networks.filter(network => !network.testnet)
+      return this.filteredNetworks.filter(network => !network.testnet)
     },
-    networks() {
+    filteredNetworks() {
       const experimentalMode = this.session.experimentalMode // "this" is not being correctly passed
       return this.networks.filter(({ enabled }) => experimentalMode || enabled)
```

Comment:
isn't this handled by the initial query already? why the filter here again?

Reply:
Nope, previously it wasn't handled